### PR TITLE
[STUB] Added Sign Up page

### PR DESCRIPTION
**Pull request by shivenmian**
@Orbiter @treba123 Added the sign up page. It's a simple static page as of now obviously. Please tell me if something more is needed in this / something is wrong.

For the authentication, **apart from the hashing**, I wanted to add some basic things: 

1. Checking if email is in a valid form (x@y.z, this can be done using regex)
2. Password == Confirm Password
3. Could also add things like Password being atleast 'x' letters long and has numbers and special chars (this again can be verified using regex)

Also we need to see if email is already taken by checking the database. 

I have left these for now as we decided on dealing with the authentication later.

I will add these too, right now it's a static page.

### CODE REVIEW

File: html/apps/signup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  src: local('Lato Regular'), local('Lato-Regular'), url(http://themes.googleusercontent.com/static/fonts/lato/v7/qIIYRU-oROkIk8vfvxw6QvesZW2xOQ-xsNqO47m55DA.woff) format('woff');
+}
+
+body {
+	background: #448ed3 ;
+	font-family: "Lato" ;
+}
+img{
+    width:15%;
+    height: 15%
```
Comment (sudheesh001) [inline html/apps/signup/css/style.css]: Codacy Issue found: [Expected end of value but found 'max-width'.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2589955281/issues/source?bid=3288751&fileBranchId=3305649#l14)

File: html/apps/signup/css/style.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	background: #d1d1d1;
+}
+.form input[type="text"] {
+	border-radius: 7px 7px 0px 0px ;
+}
+.form input[name = "pass"] {
+	border-radius: 0px 0px 0px 0px ;
+}
+.form input[name = "confirmpass"] {
+	border-radius: 0px 0px 7px 7px ;
```
Comment (sudheesh001) [inline html/apps/signup/css/style.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2589955281/issues/source?bid=3288751&fileBranchId=3305649#l62)

File: html/apps/signup/css/style.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+body {
+	background: #448ed3 ;
+	font-family: "Lato" ;
+}
+img{
+    width:15%;
+    height: 15%
+    max-width:30px;
```
Comment (sudheesh001) [inline html/apps/signup/css/style.css]: Codacy Issue found: [Expected RBRACE at line 15, col 14.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2589955281/issues/source?bid=3288751&fileBranchId=3305649#l15)

File: html/apps/signup/css/style.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+	margin: auto;
+	height: 1px ;
+	display: block;
+	background: #d1d1d1;
+}
+.form input[type="text"] {
+	border-radius: 7px 7px 0px 0px ;
+}
+.form input[name = "pass"] {
+	border-radius: 0px 0px 0px 0px ;
```
Comment (sudheesh001) [inline html/apps/signup/css/style.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2589955281/issues/source?bid=3288751&fileBranchId=3305649#l59)

File: html/apps/signup/css/style.css [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+	background: #fff;
+  	font-family:Lato ;
+  	font-weight:700 ;
+	display: block;
+	height: 45px;
+	font-size: 15px;
+	outline: none;
+	width: calc(100% - 24px) ;
+	margin: auto;
+	padding: 6px 12px 6px 12px;
```
Comment (sudheesh001) [inline html/apps/signup/css/style.css]: Codacy Issue found: [Using height with padding can sometimes make elements larger than you expect.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2589955281/issues/source?bid=3288751&fileBranchId=3305649#l41)

File: html/apps/signup/css/style.css [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+}
+.bar i {
+	width: 95%;
+	margin: auto;
+	height: 1px ;
+	display: block;
+	background: #d1d1d1;
+}
+.form input[type="text"] {
+	border-radius: 7px 7px 0px 0px ;
```
Comment (sudheesh001) [inline html/apps/signup/css/style.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2589955281/issues/source?bid=3288751&fileBranchId=3305649#l56)

**Comment by shivenmian** [comment]
Fixing the issues, wait.

**Comment by shivenmian** [comment]
@Orbiter @treba123 Please check and tell me what you think about the code and what I said.

**Comment by treba123** [comment]
Hm looks good to me. But I guess we have to come back when the other parts arrive

**Comment by Orbiter** [comment]
This looks very ugly on my screen. Please
- always start with the boilerplate app: https://github.com/loklak/loklak_server/tree/master/html/apps/boilerplate
- don't use (NEVER) other fonts than pre-defined in the current environment
- never link anyting from outside (the font loading in the css makes it possible that google can see when people are logging in in loklak)
Please fix that.

**Comment by shivenmian** [comment]
@Orbiter I started with the boilerplate. Sure, I'll use the fonts already defined.

**Pull request merged**